docs/README.md

```diff
@@ -0,0 +1,24 @@
+# Perseo documentation
+
+Perseo: Documentation
+
+## Table of Contents
+
+-   [Introduction](index.md)
+-   [Architecture](architecture/architecture.md)
+-   [Admin Guide](admin/index.md)
+-   [User Guide](user/index.md)
+-   O&M
+    -   [Deployment](admin/deployment.md)
+    -   [Configuration](admin/configuration.md)
+    -   [Administration](admin/admin.md)
+    -   [Metrics API](API/metrics_api.md)
+    -   [Logs & Alarms](admin/logs.md)
+-   Open API
+    -   [API](API/api.md)
+    -   [Actions](API/pep_actions.md)
```

Comment:
```suggestion
    -   [Available actions for PEP](API/pep_actions.md)
```

Otherwise it can be confusing (e.g. we have "actions" in Perseo that referst to what happens when a rule is triggered, and it is not the same)